gestalt/__init__.py

```diff
@@ -611,11 +596,60 @@ def fetch_vault_secrets(self) -> None:
             try:
                 secret_token = self.vault_client.secrets.kv.v2.read_secret_version(
                     mount_point=str(mount_path), path=secret_path)
+                print(secret_token)
```

Comment:
big no no

Reply:
was not planning on leaving that in? 🤣  It was still a draft for first check